src/pod/babashka/buddy.clj

```diff
@@ -173,6 +184,7 @@
     (.toString baos "utf-8")))
 
 (defn -main [& _args]
+  (Security/addProvider (BouncyCastleProvider.))
```

Comment:
I guess this isn't something we can do at compile time, outside of the -main right?

Reply:
Good to know.